cmd/ltassist/main.go

```diff
@@ -0,0 +1,176 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"sort"
+	"strconv"
+	"strings"
+
+	"github.com/mattermost/mattermost-load-test-ng/coordinator"
+	"github.com/mattermost/mattermost-load-test-ng/deployment"
+	"github.com/mattermost/mattermost-load-test-ng/loadtest"
+	"github.com/mattermost/mattermost-load-test-ng/loadtest/control"
+	"github.com/mattermost/mattermost-load-test-ng/loadtest/control/simplecontroller"
+	"github.com/mattermost/mattermost-load-test-ng/loadtest/control/simulcontroller"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	for k, c := range configs {
+		cfg, err := readDefaultConfig(k, c.defaultPath)
+		checkError(err)
+
+		c.defaultValue = cfg
+		configs[k] = c
+	}
+}
+
+var configs = map[string]config{
+	"loadtest": {
+		docPath:     "./docs/loadtest_config.md",
+		defaultPath: "./config/config.default.json",
+	},
+	"coordinator": {
+		docPath:     "./docs/coordinator_config.md",
+		defaultPath: "./config/coordinator.default.json",
+	},
+	"deployer": {
+		docPath:     "./docs/deployer_config.md",
+		defaultPath: "./config/deployer.default.json",
+	},
+	"simplecontroller": {
+		docPath:     "./docs/simplecontroller_config.md",
+		defaultPath: "./config/simplecontroller.default.json",
+	},
+	"simulcontroller": {
+		docPath:     "./docs/simulcontroller_config.md",
+		defaultPath: "./config/simulcontroller.default.json",
+	},
+}
+
+type config struct {
+	docPath      string
+	defaultPath  string
+	defaultValue control.Config
+}
+
+func main() {
+	rootCmd := &cobra.Command{
+		Use:   "ltassist",
+		Short: "Tool for load test utilities.",
+	}
+
+	configCmd := &cobra.Command{
+		Use:     "config",
+		RunE:    runConfigAssistCmdF,
+		Short:   "Create a config interactively",
+		Long:    "Interactively create selected config type and save the file.\n" + validTypes(),
+		Example: "ltassist config",
+	}
+	rootCmd.AddCommand(configCmd)
+
+	checkCmd := &cobra.Command{
+		Use:     "check",
+		RunE:    runCheckConfigsCmdF,
+		Short:   "Verify configs with the docs",
+		Long:    "Checks if the specific configs properly documented.",
+		Example: "ltassist check",
+	}
+	rootCmd.AddCommand(checkCmd)
+
+	if err := rootCmd.Execute(); err != nil {
+		os.Exit(1)
+	}
+}
+
+func runConfigAssistCmdF(_ *cobra.Command, args []string) error {
+	if len(args) > 0 {
+		if _, ok := configs[args[0]]; ok {
+			return createConfig(args[0])
+		}
+	}
+	var configNames []string
+	fmt.Printf("Pick one of the configuration type you want to create:\n")
+	for name := range configs {
+		configNames = append(configNames, name)
+	}
+	sort.Strings(configNames)
+	for i := range configNames {
+		fmt.Printf("%d. %s\n", i+1, configNames[i])
+	}
+	inp := readInput("int", "1")
+	i, err := strconv.Atoi(strings.TrimSpace(inp))
+	checkError(err)
+	if i < 1 || i > len(configNames) {
+		return fmt.Errorf("the selection must be in the range of 0 < i < %d", len(configNames))
```

Comment:
Other thing. I think  it might be better to directly exit like we do in `checkError()` otherwise we get a confusing output in addition to the actual error.